packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3669,8 +3669,6 @@ class Flex extends MultiChildRenderObjectWidget {
 /// warning box on the edge that is overflowing. If there is room on the outside
 /// of the row, the amount of overflow is printed in red lettering.
 ///
-/// {@tool sample}
```

Comment:
My understanding is that with this change the dart code in this section will no longer be processed by the analyzer (source: https://github.com/flutter/flutter/blob/4a110b6227a10637e1671f1f7ced7faf4da45d5e/dev/bots/analyze-sample-code.dart#L11-L17).

I agree that the way the sample is currently rendered in the docs is bad. However, not having this code analyzed also seems bad?

Reply:
Any ideas on how we should make it get analyzed?

Reply:
Could we wrap the {@tool ..} stuff just around the individual Dart examples in this section?

Reply:
/cc @gspencergoog since you are looking at these samples.

Reply:
Yeah, it won't get analyzed if you don't wrap it in the `{@tool `*(sample|snippet)*`}` block. It also won't get rendered as example code, or have the "copy to clipboard" button on it.

I would suggest doing what Michael suggests, and just put the "####" headings outside of the sample blocks.

Reply:
The @tool stuff really disrupts the flow of the text, I don't think it works at all well for this case.

Reply:
I could make it so all \`\`\`dart blocks get analyzed unless they contain a comment saying that they won't analyze correctly, maybe? (Haven't checked how disruptive that would be.)

Reply:
I'd like it if all `dart` blocks get analyzed by default. @gspencergoog what do you think?

Reply:
Hmm.  It's doable.  I'll have to see how disruptive it is: it could be that it works OK for all the dart blocks we have (or could work OK, with small adjustments).